dags/operators/moz_dataproc_operator.py

```diff
@@ -0,0 +1,447 @@
+import os
+import re
+import uuid
+from datetime import timedelta
+
+from airflow.contrib.hooks.gcp_dataproc_hook import DataProcHook
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.utils import timezone
+from airflow.version import version
+
+"""
+We include DataprocOperationBaseOperator(unchanged) and DataprocClusterCreateOperator here to use
+the latest master branch code, which uses the v1beta2 rest api for creating clusters.
+This is required to install optional components e.g. Jupyter/Anaconda/Component gateway.
+
+"""
+class DataprocOperationBaseOperator(BaseOperator):
+    """
+    The base class for operators that poll on a Dataproc Operation.
+    """
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 region='global',
```

Comment:
`You may notice better performance by selecting specific regions, especially based on geography, compared to the default "global" namespace.`

I think we should use `us-central1` or `us-west1`

Note: leaving this as global seems fine since we set the region to `us-central` in `moz_dataproc_runner` function.

Reply:
Yeah, and this class is pure copy paste from airflow master.  I've only modified the create operator.